### DISCUSSION TITLE
v1.35.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1325] [General]
This is mainly a bug fix release.

## Updates
- Fixed issue with `COLLECT_API_ENDPOINT` not working. Note, you need to set this at build time to work. See https://umami.is/docs/environment-variables
- Fixed chart issues in MySQL
- Updated language translations

A huge thanks to all the contributors on this release! @clem3109 @jenspots 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v1.35.0'>v1.35.0</a>.</em>

**Comment by AliMickey** [comment]
Well, the script is still hitting /api/collect.
